.eslintignore-sync

```diff
@@ -6,3 +6,4 @@ packages/{*,labs/*}/{.gitignore,.eslintignore}
 
 [inline]
 packages/localize/testdata/
+packages/localize/examples/
```

Comment:
This wasn't ignored before and the examples were causing all kinds of lint errors. Are we correctly running lint on CI? @aomarks